build-system/status-page/incident.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Sync status.amp.dev with cherry-pick progress
+ */
+
+const [number, body, before, after] = process.argv.slice(2);
+const fetch = require('node-fetch');
+const {getChannels, getFormats, steps} = require('./common');
+const {log} = require('../common/logging');
```

Comment:
If these `require` statements are separated from the `const`s below them by a blank line, lint will keep them auto-sorted.